source/features/hide-watch-and-fork-count.css

```diff
@@ -1,10 +1,5 @@
 /* The Watchers count visible (and clickable) when the dropdown is open */
 /* This isn't possible for the Forks count because of an incompatible DOM structure and #2368 */
-.rgh-hide-watch-and-fork-count:not(.rgh-forked-to) .btn-with-count[title^='Fork your own'],
-.rgh-hide-watch-and-fork-count details:not([open]) .btn-with-count[data-ga-click^='Repository, click Watch'] {
-	border-radius: 0.25em; /* Matches GitHub's default */
-}
```

Comment:
This should probably be restored

<img width="324" alt="" src="https://user-images.githubusercontent.com/1402241/86477419-af072f00-bd48-11ea-9e8a-7e6ec98937b8.png">

Reply:
😅 I probably missed that. But we should use `6px` now.